### DISCUSSION TITLE
Need help, PLEASE

### DISCUSSION THREAD

**Discussion by timcanty** [Ylianst/MeshCentral#6030] [General]
Hi, 
Anybody able to offer some help, i stupidly updated my mesh server from ubuntu 20.04 to 22.04, this removed mongo db, got in a whole heap of bother trying to get somewhere with it.

I have ended up putting mongodb version 7 on, and its failed to be access the old db files and wouldn't start.

i have got mesh working again, with a blank db, is there any way to allow the old agents to allow to connect into mesh again, or do i need to uninstall all agents and reinstall?
![image](https://github.com/Ylianst/MeshCentral/assets/32928117/a6cff28e-7442-4cc4-9ec5-30636f397501)

**Comment by dinger1986** [comment]
so did you reinstall?

**Reply by timcanty** [reply]
didn't reinstall operating system, just had to reinstall mongo, which has wiped the db, stupidly didnt have adaquote backup situation
wasn't sure if "IgnoreAgentHashCheck": true  was an option

**Reply by dinger1986** [reply]
ah shame, are you sure the db isnt still there? cause there was a way to get mongo working on 22.04 just a bit of a faff, you could try ignoreagenthashcheck, then spin up another server and install agents on there to get them more secure

**Reply by timcanty** [reply]
do you have any links to get mongo working on 22.04? i did some folder copies before i messed tomuch so i maybe able to get something back if i can find the instructions to get it working on 22.04

**Reply by timcanty** [reply]
Is there an easy way to unistall the agent via powershell, so i can then reinstall all the agents and jsut start a new db?

**Reply by dinger1986** [reply]
Yes just the name of the executable --fulluninstall

**Reply by dinger1986** [reply]
I'll have a look for the mongo threads

**Reply by dinger1986** [reply]
This might help, I'm sure it was something along these lines I used before moving to Postgres https://askubuntu.com/questions/1402179/not-able-to-install-mongodb-in-ubuntu-22-04

**Reply by timcanty** [reply]
In the end i went for a fresh install, and rerolled the agent out to all the device again.
lesson learnt the hard way, but least on the bright side we now have a nice clean install of meshcentral
